master/pkg/model/task_config_policy.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"strings"
+
+	"github.com/determined-ai/determined/proto/pkg/taskconfigpolicyv1"
+)
+
+type WorkloadType string
+
+// Constants.
+
+const (
+	// Unspecified constant.
+	UnknownType WorkloadType = "UNSPECIFIED"
+	// ExperimentType constant.
+	ExperimentType WorkloadType = "EXPERIMENT"
+	// NTSCType constant.
+	NTSCType WorkloadType = "NTSC"
+)
+
+// WorkloadTypeFromProto maps taskconfigpolicyv1.WorkloadType to WorkloadType.
+func WorkloadTypeFromProto(workloadType taskconfigpolicyv1.WorkloadType) WorkloadType {
```

Comment:
Do we also need to check whether the resulting `WorkloadType` is a valid option?

Reply:
Hmm! I was thinking about this and whether/how to test it. 
I think since the tests compare the result of`WorkloadTypeFromProto` to an expected value `test.expected`, which is always one of the pre-defined constants, that should guarantee we got the desired output.
WDYT?

Reply:
I think since the enums are typed, that it will help protect against random input. (I was trying to break the tests :) 

We would need validation in the API for user input, but that can be separate. That's input validation not type conversion. 

Thanks for thinking it through with me! I agree, it's good as it is.